src/main/java/com/cleanroommc/groovyscript/compat/mods/tinkersconstruct/Melting.java

```diff
@@ -58,6 +51,7 @@ public boolean remove(MeltingRecipe recipe) {
         return true;
     }
 
+    @MethodDescription(description = "groovyscript.wiki.removeByInput")
```

Comment:
you can remove all of the `description = "groovyscript.wiki.[methodName]"` entries.